### ISSUE TITLE
tensorflow-1-public/C1/W2/assignment/C1_W2_Assignment_Solution.ipynb  logs.get('accuracy')

### ISSUE THREAD

**Issue by gladwig2** [https-deeplearning-ai/tensorflow-1-public#11]
In the callback, the ungraded lab demonstrated:
class myCallback(tf.keras.callbacks.Callback):
  def on_epoch_end(self, epoch, logs={}):
    if(logs.get('loss')<0.4):
...

In the graded assignment the expectation seems to be that 
logs.get('accuracy')
should be used!

I suggest:
1) the ungraded and graded lab use the same method.
2) the method be 'loss' rather than accuracy because:
    a) the model.fit routine spits out loss at each iteration.
    b) accuracy has not yet been defined in the course

This will require rewriting the instructions to clarify.
Any change here will require coordination with the grader.
